### ISSUE TITLE
Enhancement:  Exit PSChiaPlotter without cancelling chia.exe's / Pause Jobs

### ISSUE THREAD

**Issue by Jaga-Telesin** [MrPig91/PSChiaPlotter#85]
Having some electrical work done at the house this afternoon, and wanted to leave the 2 (of 4) chia.exe processes running while cancelling the two that wouldn't finish in time.  If I cancel them using the GUI (or PIDs) it will try to kick off a new process due to stagger times.  If instead I close the GUI (and cancel them after using PID numbers) it will cancel all chia.exe processes on exit (is that correct?).

The message when exiting PSChiaPlotter says "Closing this window will end all Chia processes", which I interpreted to mean "we'll cancel all running plots (chia.exe processes) when exiting".  Which of course, is problematic in my situation.

- Is there perhaps a more graceful way for the exit dialog to give us the option to not cancel all jobs on exit?
- Perhaps also a way to "pause the main job" in the GUI such that it does not automatically kick off new chia.exe instances when some are terminated?  It would of course keep current runs active.

Basically looking for a touch more control over the automation and how they are terminated on exit, for those times we need to "spin down plotting in anticipation of an overall plotting break, maintenance, etc".

As a side note:  I noticed Dark Mode was integrated into the HarvesterWatcher.  Was there a plan to also integrate into ChiaPlotter?   :)

**Comment by MrPig91** [comment]
So yes, currently it will cancel all chia processes (or try to) on exit. I think the best work around for now is to pause two the two queues that have the chia processes you want to cancel and this will prevent the queue from starting another process. Or better yet, pauses all queues since you do not want to start any more processes and only want to finish the 2 you have going. 

I like both of those suggestions. I want to add them both on a future release. I also want to add dark mode, and will put it on the roadmap as well.

**Comment by Jaga-Telesin** [comment]
Good workaround, I keep forgetting to eyeball the Queue datagrid area and utilize it to better effect.

**Comment by MrPig91** [comment]
Added the ability to keep chia processes running after close in version 1.0.48. You now have the choice to end all chia processes, leave them all running or cancel the closing of the application.
